### PR TITLE
chore: update and release regularly

### PR DESCRIPTION
**Pull request by BinaryMuse**
This adds an update/release script as well as a scheduled GitHub Action to perform the release daily.

### CODE REVIEW

File: package.json
Comment (vhashimotoo) [inline package.json]: so many for semantic release, we can bump this to the latest and leave this to a single line, how in others modules?
```suggestion
    "semantic-release": "semantic-release"
```

File: script/update.sh
Comment (vhashimotoo) [inline script/update.sh:21]: why not zero?

File: script/update.sh
Comment (vhashimotoo) [inline script/update.sh]: we really create tag here? nothing to bump version, semantic-release should make this auto

File: script/update.sh
Comment (vhashimotoo) [inline script/update.sh]: 🙈 
```suggestion
git commit -am "feat: update electron-api-historian"
```

File: script/update.sh
Comment (vhashimotoo) [inline script/update.sh]: ? it's not should create one commit or it's something special case here

**Review by vhashimotoo — APPROVED** [review]
Looks mystery, and good. Some questions before this lands, otherwise ⭐.

File: script/update.sh
Comment (BinaryMuse) [reply script/update.sh:21]: 78 is the GitHub Actions "neutral" exit code, which doesn't cause a failure but also doesn't cause a green check mark. I chose it because it will allow folks to see at a glance when a job is stopping early because it's not detecting any changes.

File: script/update.sh
Comment (BinaryMuse) [reply script/update.sh]: 🙈

File: script/update.sh
Comment (BinaryMuse) [reply script/update.sh]: I'm not sure I understand your question, but this will update the repository so that the `index.json` and the sha pointed to by the submodule will match. Mostly just a bookkeeping thing.

File: package.json
Comment (BinaryMuse) [reply package.json]: Yes, I noticed this was different than the other projects too. I'm not sure why it's different, but I didn't change it. Happy to try it and see if it works. :)

File: script/update.sh
Comment (vhashimotoo) [reply script/update.sh]: The question here it's why creating 2 commits instead of 1 where one it's updated electron and where second that contains the `feat` for creating the release, with updated build data. (argh hard) easy why 1 not contain built data and updated submodule commit

> Mostly just a bookkeeping thing.

oki-doki

File: package.json
Comment (vhashimotoo) [reply package.json]: > Happy to try it and see if it works. :)

Let's go on wheels of experiments and tries

File: script/update.sh
Comment (BinaryMuse) [reply script/update.sh]: Gotcha, yes that makes sense

File: package.json
Comment (BinaryMuse) [reply package.json]: This might be because our version of semantic-release is very old. I'll bump it as part of this PR.

**Review by vhashimotoo — APPROVED** [review]
🏍🛥🏁

**Pull request merged**
